workflow-bwa-mem/workflow/config/workflow.ini

```diff
@@ -32,17 +32,18 @@ do_trim=false
 # key=trim_mem_mb:type=integer:display=F
 trim_mem_mb=16384
 # key=trim_min_quality:type=integer:display=F
-trim_min_quality=20
+trim_min_quality=0
 # key=trim_min_length:type=integer:display=F
-trim_min_length=10
+trim_min_length=1
 cutadapt_r1_other_params=
 cutadapt_r2_other_params=
-r1_adapter_trim=AGATCGGAAGAGCGGTTCAGCAGGAATGCCGAGACCG
+r1_adapter_trim=AGATCGGAAGAGCACACGTCTGAACTCCAGTCACACGCTCGAATCTCGTA
 r2_adapter_trim=AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT
 
 # Sorry to non-OICR folks, but I was falling down a rabbit hole of required dependencies
 python=/oicr/local/sw/Python-2.7.2/bin/python2.7
-cutadapt=/oicr/local/analysis/sw/cutadapt/cutadapt-1.3/bin/cutadapt
+# USE CUTADAPT VERSION1.8.3
+cutadapt=/oicr/local/analysis/sw/cutadapt/1.8.3/bin/cutadapt
```

Comment:
Oh, if you do this you don't need module load.

Reply:
I have tried running it on the cluster, and it will complain if the cutadapt/1.8.3 module is not loaded before calling cutadapt ( the error message is `Traceback (most recent call last): File "/oicr/local/analysis/sw/cutadapt/1.8.3/bin/cutadapt", line 9, in <module>
    from cutadapt.scripts import cutadapt
ImportError: No module named cutadapt.scripts`). Cutadapt1.3 does not have this issue though. It seems like an issue particularly with cutadapt1.8.3. I moved `module load cutadapt/1.8.3` from the java file to `workflow.ini`, so if people do not want any module to be loaded, they can modify `workflow.ini` and empty the `module` property without needs of changing the workflow directly.